krausening-python/pom.xml

```diff
@@ -18,6 +18,9 @@
             <plugin>
                 <groupId>org.technologybrewery.habushu</groupId>
                 <artifactId>habushu-maven-plugin</artifactId>
+                <configuration>
+                    <pythonVersion>${version.python.default}</pythonVersion>
+                </configuration>
             </plugin>
```

Comment:
Q: Will this change the value of the current version of the project in the [pyproject.toml](https://github.com/TechnologyBrewery/krausening/blob/d297c64416d4a72d6352d5983a88a61804207d33/krausening-python/pyproject.toml#L3C1-L3C19)?

Reply:
No - this is the version that we'll use to build only.